region/maridia/inner-pink/Crab Shaft.json

```diff
@@ -1918,6 +1983,45 @@
         "Knocking the crab off while it is on the the door or ceiling will not work."
       ]
     },
+    {
+      "link": [4, 4],
+      "name": "Sciser Farm (1 Sciser, Fast Cycle)",
+      "requires": [
+        {"resetRoom": {"nodes": [3]}},
+        {"or": [
+          "canCrouchJump",
+          "h_underwaterCrouchJumpWithFlashSuit"
+        ]},
+        {"or": [
+          {"and": [
+            "Plasma",
+            {"cycleFrames": 150}
+          ]},
+          {"and": [
+            "Wave",
+            "Spazer",
+            {"cycleFrames": 160}
+          ]},
+          {"and": [
+            "Grapple",
+            {"cycleFrames": 160}
+          ]}
+        ]},
+        {"or": [
+          "Grapple",
```

Comment:
I can't pull off a grapple kill and stay on the platform.

Reply:
There aren't any instant kill beams listed, so I don't see grapple helping in any other case either.

Reply:
> I can't pull off a grapple kill and stay on the platform.

I was doing it from a water entry above. It looks like for it to work from an air entry, it needs either a wall jump or Morph to reset fall speed. Since we don't yet have a way to represent a water entry, I can just add requirements for one of those two things.